docs/example/saxml/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: saxml-model
+data:
+  publish.conf: |
+    model=/sax/test/spmd1
+    model_path=saxml.server.pax.lm.params.lm_cloud.LmCloudSpmd175B32Test
+    checkpoint=None
```

Comment:
I guess this is is none because we are using a test model, lets replace that with an actual one, gemma and llama for example.

Reply:
Correct, it is none because it is a test model. I'll see if there are any other models that are supported for multihost. As far as I'm aware, Saxml only supports single host for Gemma and llama (and the topology is different as well)

Reply:
I thought it supports llama2 70B, but lets leave this for a followup because we will need to add instructions on how to download and convert the image.